src/elements-experimental/seat-reservation/seat-reservation/seat-reservation-base-element.ts

```diff
@@ -34,6 +34,8 @@ interface CoachScrollTriggerPoint {
   width: number;
 }
 
+export type SeatReservationSelectedPlacesEventDetails = SeatReservationPlaceSelection[];
```

Comment:
Is this type really necessary?

Reply:
Probably not, but the react wrapper generator fails with an exception and this was the only work around I found :D. the same for another event details where I had to assign "number" which is even more strange. But looking at the generator script, I couldn't find a solution. Maybe you have an idea?

Reply:
Good question. I think it is fine for now.